crates/bevy_tasks/src/task_pool.rs

```diff
@@ -34,6 +36,7 @@ pub struct TaskPoolBuilder {
     /// If set, we'll set up the thread pool to use at most `num_threads` threads.
     /// Otherwise use the logical core count of the system
     num_threads: Option<usize>,
+    num_blocking_threads: Option<usize>,
```

Comment:
There should be a function on builder to set `num_blocking_threads`

Reply:
Done!